packages/swap/src/utils/screenAddress.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import { z } from 'zod';
+
+const schema = z.object({ identifications: z.array(z.object({})) });
+
+export default async function screenAddress(address: string): Promise<boolean> {
+  const apiKey =
+    process.env.CHAINALYSIS_API_KEY ??
+    '8d6abb8da3576baec39e261f060231ae7753034bfba92e5044dc1effe4251c1f';
+
+  if (!apiKey) return false;
+
+  const response = await axios.get(
+    `https://public.chainalysis.com/api/v1/address/${address}`,
+    { headers: { 'X-API-Key': apiKey } },
```

Comment:
Maybe having a try catch here wouldn't hurt so that their service interruptions don't affect us

Reply:
yes i did intend to do that

Reply:
done